Makefile

```diff
@@ -657,5 +657,53 @@ ifeq ($(CI),)
 src/interface_py/h2o4gpu/BUILD_INFO.txt: .ALWAYS_REBUILD
 endif
 
+#----------------------------------------------------------------------
+# CentOS 7 build API BEGIN
+#
+# Summary
+#
+#     command:  make centos7_in_docker
+#     output:   dist/h2o4gpu*.whl
+#
+# Details
+#
+#     This is only supported in a docker environment.
+#
+#     The 'centos7' make target does the actual work.
+#
+#     The 'centos7_in_docker' make target sets up the docker environment
+#     and then invokes the work inside that environment.
+#
+#     The build output is put in the 'dist' directory in h2o4gpu level.
+#----------------------------------------------------------------------
+
+centos7_setup:
+	rm -fr /tmp/build
+	cp -a /dot/. /tmp/build
+
+centos7_build:
+	(cd /tmp/build && \
+         eval "$$(/root/.pyenv/bin/pyenv init -)" && \
+         /root/.pyenv/bin/pyenv global 3.6.1 && \
+         export IFLAGS="-I/usr/include/openblas" && \
+         export OPENBLAS_PREFIX="open" && \
+         scl enable devtoolset-3 "make fullinstalljenkins-nonccl-cuda8")
+	cp /tmp/build/src/interface_py/dist2/h2o4gpu*.whl dist
+	chmod o+rw dist/h2o4gpu*.whl
+
+centos7:
+	$(MAKE) centos7_setup
+	$(MAKE) centos7_build
+
+centos7_in_docker:
```

Comment:
These makefile additions mean there is a separate path for centos compared to before.  If we decide centos is a way to go to get a base compatible version, everything should be redone like originally with scripts for different builds (nccl, cuda, etc.)